### PR TITLE
Add The Weather Network

### PR DESCRIPTION
**Pull request by soodaayush**

### CODE REVIEW

File: img/t/theweathernetwork.com.svg [hunk 1]
```diff
@@ -0,0 +1 @@
....4-31.3 45.7-38.2-5.1 12.4-29.4 31.7-45.7 38.2zm42-46.4c3.1 0 4.7 1.7 4.7 4.1 0 .7-.1 1.5-.3 2.2-16.1 6.9-37.6 24.4-52.6 42-1 .2-2 .3-3 .3-2.3.2-4.7-.4-6.7-1.5 11.5-20 43.9-47.1 57.9-47.1zm-61.8 62.7c-.3-1.1-.5-2.2-.4-3.3.1-2.4.7-4.8 1.7-7.1 1.6.8 3.5 1.2 5.3 1.1 1 0 2.1-.1 3.1-.2-2.4 3.1-4.6 6.3-6.6 9.6-.5.9-1.1 1.8-1.5 2.7-.8-.9-1.3-1.8-1.6-2.8zm8.1 23.1c-4.5-1.5-5.2-5.2-5.2-8.1 0-1.4.2-2.8.6-4.2.8.1 1.7.2 2.5.2 3.1-.1 6.2-.6 9.2-1.5-2.8 4.2-5.2 8.8-7.1 13.6zM60.1 365c-5.6-.4-8.1-3.8-8.1-8.6v-1.7c4.9-.6 9.6-1.9 14.2-3.7-2.7 4.5-4.7 9.2-6.1 14zm17 12.3h-1.8c-5.9 0-8.8-2.9-9.6-7.2 5.1-1.1 10.1-2.7 14.9-4.8-1.9 3.8-3.1 7.8-3.5 12zm20.5 5.8c-1.6.4-3.2.6-4.8.6-3.1.1-6.1-1.1-8.2-3.4 4.8-1.7 9.4-3.8 13.8-6.2-.7 2-1 4.1-1 6.3 0 .9.1 1.8.3 2.6v.1zM83 376.3v-.5c.3-4.7 1.7-9.4 4.1-13.5 18.2-9.5 34.7-21.9 48.8-36.8 5.2-5.3 9.6-11.3 13.2-17.8.9-.2 1.9-.3 2.9-.3 5 0 5.8 3.6 5.8 6 0 3.3-1.8 7.4-4.8 11.8-.6.1-1.3.2-1.9.4-14.8 3.4-41.5 24.9-50.6 43.3-5.5 3.1-11.4 5.6-17.5 7.4zm68.8-49.5c-9.2 12.6-27.6 27.9-45 38.4 8.9-15.9 31.9-35.2 45-38.4zm-48.7 49.8c0-2 .4-4.1 1.2-6 19.8-11.8 40.4-29.9 49.8-44.3h1.1c2.3-.2 4.3 1.4 4.5 3.6v.2c0 13.1-33.3 42.2-55.8 50.8-.6-1.3-.8-2.8-.8-4.3z" class="st0"/></svg>
```
Comment (hkamran80) [inline img/t/theweathernetwork.com.svg]:
```suggestion
<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 140.8 143.3"><path fill="#fee953" d="M15.4 0h110c8.5 0 15.4 6.9 15.4 15.4v112.5c0 8.5-6.9 15.4-15.4 15.4h-110c-8.5 0-15.4-6.9-15.4-15.4V15.4C0 6.9 6.9 0 15.4 0Z"/><path fill="#1f3d73" d="M83.9 130.8a10 10 0 0 1-3.4-.5c24.4-11.8 54-40.3 54-53.7a5 5 0 0 0-3.5-5c-.9-.3-1.8-.5-2.7-.5 2.4-3.5 3.9-7.5 4.3-11.7.4-3.7-2.3-7-6-7.4h-1.4l-1.9.2c1.3-2.6 2.1-5.5 2.2-8.5 0-5.4-3.6-7.7-8.4-7.7-1.9 0-3.7.3-5.5.7 1.2-2.3 1.8-4.9 1.9-7.5 0-4.1-2.2-7.5-9-7.5-2.3 0-4.5.4-6.7 1 .5-1.2.7-2.5.7-3.8 0-4.5-2.9-6.7-7.7-6.7a31 31 0 0 0-10.4 2.4l.1-1.4c0-3.6-1.9-5.5-6.4-5.5C59.2 7.7 28.7 31 14 52.2l.1-1.7c0-8.1 10.3-26.7 36.3-38.7l-.7-1.4C22.2 21.8 9 43.6 9 52.7c0 1.9.5 3.9 1.5 5.5A31.3 31.3 0 0 0 6 71.8v.5a9.6 9.6 0 0 0 5.2 8.8c-1.3 3-2 6.1-2.1 9.4.2 5.6 4.5 10.2 10.1 10.9a20 20 0 0 0-.4 4.2 11.5 11.5 0 0 0 11.1 11.9h2.6v.5c0 7.7 5.3 11.2 12.2 11.2 2.1 0 4.2-.3 6.2-.7 1.6 5.2 7.2 7.4 11.2 7.4 3.8-.1 7.6-.9 11.2-2.3 2.2 1.7 4.9 2.5 7.7 2.3 12 0 35.6-14.6 46.2-33.4l-1-1C113.3 121 93 130.8 83.9 130.8Zm33.4-93.6a6.4 6.4 0 0 1 7 5.7v.9c-.2 3.1-1.1 6.1-2.7 8.8-8 1.9-20.5 9.3-32.7 19.1-12.3 9.8-24 21.8-31.3 33.5a72.3 72.3 0 0 1-18.3 5.8 49.1 49.1 0 0 1 7-16.3 166.6 166.6 0 0 0 35.2-23A120.7 120.7 0 0 0 111 38.4c2-.8 4.2-1.1 6.3-1.2zm3.2 17.1c-4.1 6.4-9 12.2-14.5 17.3a192.5 192.5 0 0 1-41.3 29.9c8-11.2 17.4-21.3 28.1-30a98 98 0 0 1 27.7-17.2zM50.9 87.8a165 165 0 0 1 13.9-16.2C78.9 57 96.1 43.9 108.6 39.2a133.1 133.1 0 0 1-32.1 32.4 170 170 0 0 1-25.6 16.2Zm54.4-64.7c4.8 0 6.9 2.4 6.9 6-.2 2.9-1.1 5.7-2.5 8.2-14 4.2-33.4 18.2-49.3 34.3-6 6-11.6 12.6-16.5 19.5a48.4 48.4 0 0 1-17.2 4.3c1.9-6 4.7-11.6 8.2-16.8 4.7-2 9.2-4.4 13.5-7.1 21.1-12.8 43.3-33.9 49.1-47.1 2.4-.8 5.1-1.2 7.8-1.3zM39.1 72.4l.6-.8c14.9-19.8 38.5-40.1 55.8-46.5-7.1 13.1-32.7 36.1-54.6 46.5Zm52-58.9c4.8 0 6.2 1.2 6.2 5.8-.1 1.3-.4 2.6-.9 3.8C78.2 28.7 51.8 50.3 35 71.5l-2.6 3.4c-3.4 1.2-7 1.8-10.5 1.9l-3.3-.3c.7-1.6 1.6-3.2 2.5-4.9a115 115 0 0 1 7.3-10.9c20.4-6.6 47.3-31.6 51.8-44.4 3.5-1.6 7.2-2.6 10.9-2.8zM32.8 55.3a159.7 159.7 0 0 1 45.7-38.2c-5.1 12.4-29.4 31.7-45.7 38.2Zm42-46.4c3.1 0 4.7 1.7 4.7 4.1 0 .7-.1 1.5-.3 2.2-16.1 6.9-37.6 24.4-52.6 42l-3 .3a12 12 0 0 1-6.7-1.5C28.4 36 60.8 8.9 74.8 8.9ZM13 71.6a9.3 9.3 0 0 1-.4-3.3 20 20 0 0 1 1.7-7.1c1.6.8 3.5 1.2 5.3 1.1l3.1-.2c-2.4 3.1-4.6 6.3-6.6 9.6l-1.5 2.7a7.2 7.2 0 0 1-1.6-2.8zm8.1 23.1c-4.5-1.5-5.2-5.2-5.2-8.1 0-1.4.2-2.8.6-4.2l2.5.2c3.1-.1 6.2-.6 9.2-1.5a73.1 73.1 0 0 0-7.1 13.6zm12.5 16.4c-5.6-.4-8.1-3.8-8.1-8.6v-1.7c4.9-.6 9.6-1.9 14.2-3.7a59.5 59.5 0 0 0-6.1 14zm17 12.3h-1.8c-5.9 0-8.8-2.9-9.6-7.2a78.5 78.5 0 0 0 14.9-4.8 33.8 33.8 0 0 0-3.5 12zm20.5 5.8c-1.6.4-3.2.6-4.8.6-3.1.1-6.1-1.1-8.2-3.4 4.8-1.7 9.4-3.8 13.8-6.2a18.9 18.9 0 0 0-.7 8.9v.1zm-14.6-6.8v-.5c.3-4.7 1.7-9.4 4.1-13.5a185 185 0 0 0 48.8-36.8 82.2 82.2 0 0 0 13.2-17.8c.9-.2 1.9-.3 2.9-.3 5 0 5.8 3.6 5.8 6a23 23 0 0 1-4.8 11.8l-1.9.4C109.8 75.1 83.1 96.6 74 115a84.4 84.4 0 0 1-17.5 7.4zm68.8-49.5a169.2 169.2 0 0 1-45 38.4c8.9-15.9 31.9-35.2 45-38.4zm-48.7 49.8c0-2 .4-4.1 1.2-6 19.8-11.8 40.4-29.9 49.8-44.3h1.1a4 4 0 0 1 4.5 3.6v.2c0 13.1-33.3 42.2-55.8 50.8-.6-1.3-.8-2.8-.8-4.3z"/></svg>
```

File: entries/t/theweathernetwork.com.json [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+  "The Weather Network": {
+    "domain": "theweathernetwork.com",
+    "contact": {
+      "twitter": "weathernetwork",
+      "facebook": "theweathernetwork",
+      "form": "https://help.theweathernetwork.com/hc/en-us/requests/new"
+    },
+    "categories": [
+      "other"
+    ]
```
Comment (hkamran80) [inline entries/t/theweathernetwork.com.json:22]:
```suggestion
    ],
    "regions": [
      "au",
      "ca",
      "de",
      "es",
      "fr",
      "gb",
      "ie",
      "in",
      "us"
    ]
```

Selected from the country switcher

**Review by hkamran80 — CHANGES_REQUESTED** [review]

**Comment by soodaayush** [comment]
Changes made.

**Review by hkamran80 — APPROVED** [review]

**Comment by hkamran80** [comment]
Thank you! Your contribution will be visible in around 10 minutes, although it may take longer due to server and browser caches.

**Pull request merged**